### ISSUE TITLE
[BUG REPORT] Can hear anyone communication voice anywhere on the map while use TOR mod on Linux.

### ISSUE THREAD

**Issue by showBeginner** [OhMyGuus/BetterCrewLink#330]
### Pre-Flight Checklist

- [X] I am not asking a question => use the [Discord](https://discord.gg/qDqTzvj4SH) if you are
- [X] I have tried to use an [alternative voice server](https://bettercrewl.ink/)
- [X] I have checked everyone in my lobby is on the same BetterCrewLink/CrewLink server
- [X] I have refreshed and activated <kbd>NAT Fix</kbd> on the BetterCrewLink app when I can't hear some people (this is a known issue)
- [X] I have checked that the BetterCrewLink/CrewLink server I'm using is up to date
- [ ] I have a screenshot or videos of any errors
- [X] I have checked that someone else has not reported this using the [search bar](https://github.com/OhMyGuus/BetterCrewLink/issues?q=is%3Aissue)

### Describe the bug

My friend happened #315 issue so  use ` --disable-gpu-sandbox ` flag to start app . 
Though Fix #315 issue but he can hear anyone communication voice anywhere on the map.

### BetterCrewLink:
```
version: 3.0.3 
```
### Install  & Run Command:
```
wget "https://mirror.bettercrewl.ink/pc/latest/BetterCrewlink-Linux.AppImage"
chmod +x BetterCrewlink-Linux.AppImage
sudo apt-get install libcap2-bin 
sudo setcap cap_sys_ptrace=eip /usr/bin/wineserver
./BetterCrewlink-Linux.AppImage --disable-gpu-sandbox
```

### Steps to reproduce the behavior

```
1. ./BetterCrewlink-Linux.AppImage --disable-gpu-sandbox
2. start the among us game with friend.
3. issue happened
```

### Expected behavior

He only can hear nearby people sound. 

### Desktop

- OS: Ubuntu 
- Version: [ 22.04 ]

**Comment by OhMyGuus** [comment]
Try our nightly version it has a lot of fixes for Linux (which I still need to release)
https://nightly.link/OhMyGuus/BetterCrewLink/suites/7807971475/artifacts/329541186

**Comment by gevaerts** [comment]
I had the same issue last week. I'll try the nightly soon.

**Comment by gevaerts** [comment]
I tried the nightly in a lobby (no full game yet) with the latest TOR installed (TOR 4.1.7 on Among Us 2022.8.24) and that seemed to work. I don't get the warning icons anymore (I got a red icon with a white exclamation mark for everyone in the BCL window last week), and I do get directional effects in the lobby.

I'll be able to test with an actual game (i.e. enough other players to actually start the game) tomorrow. I'll report back then, but it's looking good so far.
